### PR TITLE
Navbar and techspeakers

### PR DESCRIPTION
**Pull request by baamaboi**

### CODE REVIEW

File: gallery_events.html [hunk 1]
```diff
@@ -52,7 +53,7 @@
 
       <div class="col-6 col-xl-2" data-aos="fade-down">
         <!-- <h1 class="mb-0"><a href="index.html" class="text-black h2 mb-0">SPEC</a></h1>-->
-        <a href="index.html"><img src="gallery\bg-img\spec_logo.png" class="logo" /></a>
+        <a href="index.html"><img src="imgs\headicon.ico" class="logo" /></a>
```
Comment (hiteshsandal) [inline gallery_events.html]: @baamaboi, is this working fine with .ico file?

Comment (baamaboi) [reply gallery_events.html]: yes

**Comment by sjainit** [comment]
@baamaboi , few changes:

1. In the navbar of Electrothon page, add 'Speakers'. 
2. In the gallery section, the images on the opening page need to be changed as told before.
3. Also, the speakers section added needs to formatted. Try to add the hovering effect on the images in the speakers section as on Team SPEC page. Add their bio and topic of the talk.

**Comment by baamaboi** [comment]
ok will do

**Comment by sjainit** [comment]
@baamaboi, few more tweaks:
1.) The text in the speakers' section(which appears on hovering), needs to be more structured. Also, add the LinkedIn and Facebook profiles of speakers.

**Comment by sjainit** [comment]
Good work @baamaboi. I will close this PR, as the speakers' section is urgent. For the blogs section, we need to open a new PR.

**Pull request merged**